integration/fuse/bin/alluxio-fuse.sh

```diff
@@ -12,7 +12,7 @@
 
 SCRIPT_DIR="$(cd "$(dirname "$(readlink "$0" || echo "$0")")"; pwd)"
 
-get_env () {
+function getEnv {
```

Comment:
how about following google bash style guide in naming?
https://google.github.io/styleguide/shell.xml#Function_Names

Reply:
I'm fine with the proposed style, but I changed it to be consistent with the style in `bin/alluxio`. 
In this case, I'll change the styles of all our scripts.